src/styles/developer-tools.css

```diff
@@ -0,0 +1,11 @@
+.developer-tools-page {
```

Comment:
Sor, just thought of this.

Should we move `developer-tools.css` to `src/components/DeveloperToolCard`?

Maybe we should also refactor existing codebase to be component-isolated too. But that's another issue

Reply:
@pkong-ds but this is for the page, not the `DeveloperToolCard` component, that's why I didn't move it to `src/components/DeveloperToolCard`, how about renamed the folder to `src/components/DeveloperTools` and put  `developer-tools.css` in?

Reply:
@YayunHuang I am thinking below. Any thoughts?

```
src/
   pages/
       DeveloperToolPage/
           DeveloperToolPage.astro
           developer-tool-page.css
           DeveloperToolCard.astro
       // later move all pages here. one page one folder
```

Reply:
sound nice!! let me update it and create an issue for other pages~~

Reply:
@pkong-ds rebased to main and update, but the script must put in `public/scripts`, so I only move `.astro` and `.css` file~

Reply:
ah, scripts import 😞 ref https://linear.app/oursky/issue/MUP-107/[code-quality]-refactoring

Will merge this first then 🙏